src/main/java/util/xcmailr/XcMailrHelper.java

```diff
@@ -16,21 +16,23 @@
 public class XcMailrHelper
 {
     /**
-     * Retrieve messages's text content from json response body
+     * Retrieves the text content of the first e-mail from the JSON response body. *
      * 
      * @param responseBody
-     * @return messages's text content
+     *            is the return value of XcMailApi.fetchEmails() and its sugar methods
```

Comment:
Generally, the Javadoc comment of a parameter starts with the type of the parameter and then describe what it is. It should also not be too technical, e.g.

```suggestion
     *            a String containing the response of an XcMailr API call, e.g. <code>XcMailrApi.fetchEmails</copde>
```